### PR TITLE
Hw25

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

**Review by pavlokhomiak — APPROVED** [review]

File: src/main/java/ecommerce/dao/interfaces/ProductDao.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package ecommerce.dao.interfaces;
+
+import ecommerce.dao.GenericDao;
+import ecommerce.model.Product;
+
+public interface ProductDao extends GenericDao<Product, Long> {
+
+
```
Comment (RomanDubovskyi) [inline src/main/java/ecommerce/dao/interfaces/ProductDao.java]: redundant empty lines

File: src/main/java/ecommerce/dao/interfaces/UserDao.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package ecommerce.dao.interfaces;
+
+import ecommerce.dao.GenericDao;
+import ecommerce.model.User;
+
+public interface UserDao extends GenericDao<User, Long> {
+
```
Comment (RomanDubovskyi) [inline src/main/java/ecommerce/dao/interfaces/UserDao.java]:
```suggestion
```

File: src/main/java/ecommerce/service/interfaces/ProductService.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package ecommerce.service.interfaces;
+
+import ecommerce.model.Product;
+import ecommerce.service.GenericService;
+
+public interface ProductService extends GenericService<Product, Long> {
+
```
Comment (RomanDubovskyi) [inline src/main/java/ecommerce/service/interfaces/ProductService.java]:
```suggestion
```

**Review by RomanDubovskyi — COMMENTED** [review]
all good, just remove redundant empty lines

File: src/main/java/ecommerce/dao/interfaces/ProductDao.java (hunk 1 above)
Comment (tarkaiev) [reply src/main/java/ecommerce/dao/interfaces/ProductDao.java]: fixed

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**